docs/OperationsCommittee.md

```diff
@@ -15,6 +15,15 @@ title: Operations Committee
 The members of the Operations Committee are listed [here](Membership.html).
 See a description of our activities in our [mission statement](MissionStatement.html).
 
+## OBO Operations Committee SOP
+[Standard operating procedures (SOPs)](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/docs/SOP.md) for the OBO Foundry Operations Committee can be found in our GitHub repo.
+That document describes:
+- New Ontology Requests
+- Reviewing Ontologies for OBO Membership
+- Ontology acceptance email
+- Changing ontology metadata in the registry
+- Chairing an OBO Operations Committee meeting call
```

Comment:
We would have to keep this list up to date whenever SOP.md changes.. Also would be great to add SOP to menu somewhere.

Reply:
I see it now in the About menu -- did you just add it?